app/controllers/admin/contacts_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ContactsController < ApplicationController
+  before_action :set_contact, only: [:show, :edit, :update, :destroy]
+
+  # GET /contacts
+  # GET /contacts.json
+  def index
+    @contacts = Contact.all
+  end
+
+  # GET /contacts/1
+  # GET /contacts/1.json
+  def show
+  end
+
+  # GET /contacts/new
+  def new
+    @contact = Contact.new
+  end
+
+  # GET /contacts/1/edit
+  def edit
+  end
+
+  # POST /contacts
+  # POST /contacts.json
+  def create
+    @contact = Contact.new(contact_params)
+
+    respond_to do |format|
+      if @contact.save
+        format.html { redirect_to @contact, notice: 'Contact was successfully created.' }
+        format.json { render :show, status: :created, location: @contact }
+      else
+        format.html { render :new }
+        format.json { render json: @contact.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /contacts/1
+  # PATCH/PUT /contacts/1.json
+  def update
+    respond_to do |format|
+      if @contact.update(contact_params)
+        format.html { redirect_to @contact, notice: 'Contact was successfully updated.' }
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [SideCI](https://sideci.com) -->